src/base/components/FeedList.js

```diff
@@ -96,8 +96,7 @@ class FeedListPage extends React.Component {
   }
 
   setConfig = () => {
-    var setval={dev:"Mote",opr:this.state.devicestatus};
-    console.log("Here object is "+JSON.stringify(object));
+    var setval={dev:"Mote",opr:this.state.devicestatus};    
     fetch("https://immense-temple-14028.herokuapp.com/api/Device", {
```

Comment:
Hardcoded

Reply:
its not hardcoded, i just removed one line which created exception